src/messages/GE.json

```diff
@@ -295,7 +295,9 @@
   "see more queries":"See more queries in the",
   "Microsoft Graph API Reference docs": "Microsoft Graph API Reference docs.",
   "Fetching permissions": "Fetching permissions",
-  "view all permissions": "View all permissions",
+  "view all permissions": "Select permissions",
   "Fetching code snippet": "Fetching code snippet",
-  "Snippet not available": "Snippet not available"
+  "Snippet not available": "Snippet not available",
+  "Select different permissions": "To try out Microsoft Graph API endpoints, select different permissions then click Consent",
```

Comment:
i like how classic links to the permissions page which gives you a little bit better idea of what messing with the different permissions will do.

Reply:
I can add the link to the permissions page on the docs

Reply:
not blocking this PR for me.  Can be done on another change

Reply:
@ddyett, it's not called classic, it's **v3**. :). Welcome to my TED talk.